### PR TITLE
Update html and image generation from notebooks with tests against expectations

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

File: nb/projects/gear/gear.nb [hunk 1]
```diff
@@ -19,5 +19,4 @@ const example = Gear()
   .hasToothResolution(toothResolution)
   .ex(thickness)
   .material('wood')
-  .view(1024)
```
Comment (BarbourSmith) [inline nb/projects/gear/gear.nb:22]: I know this is a deleted line, but what does the argument to view() do?

**Review by BarbourSmith — APPROVED** [review]
I don't have any suggestions for changes, but I did learn some things by reading through and I'm even more impressed with this system now than I was before. Looks good to me.

File: nb/projects/gear/gear.nb (hunk 1 above)
Comment (pentacular) [reply nb/projects/gear/gear.nb:22]: It was a size at one time. :)

Comment (BarbourSmith) [reply nb/projects/gear/gear.nb:22]: Ah right, I remember that now.

**Pull request merged**